downstream/modules/platform/proc-aap-hub-backup.adoc

```diff
@@ -0,0 +1,26 @@
+[id="aap-hub-backup"]
+
+= Backing up the {HubNameStart} deployment
+
+[role=_abstract]
+Use this procedure to back up a deployment of the hub, including all hosted Ansible content.
+
+.Prerequisites
+
+* You must be authenticated with an Openshift cluster.
+* The {OperatorPlatform} has been installed to the cluster.
+* The {HubName} is deployed to using the {OperatorPlatform}.
+
+.Procedure
+. Log in to *{OCP}*.
+. Navigate to menu:Operators[Installed Operators].
+. Select the {OperatorPlatform} installed on your project namespace.
+. Select the *Automation Hub Backup* tab.
```

Comment:
Same question as above in  controller backup section. Should this follow the accessibility tip or is the tip only for buttons or other UI elements?

```suggestion
. Select the btn:[Automation Hub Backup] tab.
```

Reply:
> Same question as above in controller backup section. Should this follow the accessibility tip or is the tip only for buttons or other UI elements?

No, see response above for explanation.